src/pages/steps/OrganizerPicker.tsx

```diff
@@ -1,75 +1,124 @@
 };
 
 const OrganizerPicker = ({
-  value,
+  organizer,
   onChange,
   onAddNewOrganizer,
+  onDeleteOrganizer,
   ...props
 }: Props) => {
-  const { i18n, t } = useTranslation();
+  const { t, i18n } = useTranslation();
+  const queryClient = useQueryClient();
 
+  const [addButtonHasBeenPressed, setAddButtonHasBeenPressed] = useState(false);
   const [organizerSearchInput, setOrganizerSearchInput] = useState('');
 
-  const getOrganizersQuery = useGetOrganizersQuery(
+  const getOrganizersByQueryQuery = useGetOrganizersByQueryQuery(
     { q: organizerSearchInput },
     { enabled: !!organizerSearchInput },
   );
 
   const organizers = useMemo(() => {
     // @ts-expect-error
-    return getOrganizersQuery.data?.member;
+    return getOrganizersByQueryQuery.data?.member ?? [];
     // @ts-expect-error
-  }, [getOrganizersQuery.data?.member]);
+  }, [getOrganizersByQueryQuery.data?.member]);
 
   return (
     <Stack {...getStackProps(props)}>
       <FormElement
         id="create-organizer"
         label={t('create.additionalInformation.organizer.title')}
         Component={
-          <Typeahead<Organizer>
-            options={organizers}
-            labelKey={(organizer) => {
-              if (typeof organizer.name === 'string') {
-                return organizer.name;
-              }
-
-              return (
-                organizer.name[i18n.language] ??
-                organizer.name[organizer.mainLanguage]
-              );
-            }}
-            selected={value ? [value] : []}
-            onInputChange={debounce(setOrganizerSearchInput, 275)}
-            onChange={(organizers) => {
-              const organizer = organizers[0];
+          organizer ? (
+            <Inline
+              justifyContent="space-between"
+              alignItems="center"
+              paddingY={3}
+              spacing={3}
+            >
+              <Text>
+                {getOrganizerName(
+                  organizer,
+                  i18n.language as Values<typeof SupportedLanguages>,
+                )}
+              </Text>
+              <Button
+                spacing={3}
+                variant={ButtonVariants.SECONDARY}
+                onClick={() =>
+                  onDeleteOrganizer(parseOfferId(organizer['@id']))
+                }
+              >
+                {t('create.additionalInformation.organizer.change')}
+              </Button>
+            </Inline>
+          ) : !addButtonHasBeenPressed ? (
+            <Button
+              variant={ButtonVariants.SECONDARY}
+              onClick={() => setAddButtonHasBeenPressed(true)}
+            >
+              {t('create.additionalInformation.organizer.add_new_button')}
+            </Button>
+          ) : (
+            <Typeahead<Organizer>
+              options={organizers}
+              labelKey={getLabelKey(
+                i18n.language as Values<typeof SupportedLanguages>,
+              )}
```

Comment:
You could use an arrow function here I guess
```suggestion
              labelKey={(org: Organizer) => getOrganizerName(org, language)}
```